concurrencykata/src/main/java/kata/concurrency/testframework/internal/model/TestContext.java

```diff
@@ -0,0 +1,30 @@
+package kata.concurrency.testframework.internal.model;
+
+public class TestContext {
+
+    private final TestGivenContext givenContext;
+    private final TestWhenContext whenContext;
```

Comment:
[question] why dont we have TestThenContext? would be nice matching of the domain

Reply:
[answer] at first i've created this class but based on the superfluity it has been deleted.